recipes/flatten-dict/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "flatten-dict" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 0ccc43f15c7c84c5ef387ad19254f6769a32d170313a1bcbf4ce582089313d7e
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - poetry
+  run:
+    - pathlib2 >=2.3,<3.0
```

Comment:
it looks like you need `setuptools` under `run` due to importing `pkg_resources`